### ISSUE TITLE
Mermaid: Uncaught SyntaxError

### ISSUE THREAD

**Issue by nodeg** [thegeeklab/hugo-geekdoc#229]
I am not able to use Mermaid. I created a sample page and copied the content from the [documentation page](https://geekdocs.de/shortcodes/mermaid/) for testing purposes. I did not use the repository but the pre-built release bundle (v0.19.4). Furthermore I used your [example configuration](https://geekdocs.de/usage/configuration/) and rendered the page the following way:

```bash
$ hugo server -D
Start building sites … 
hugo v0.89.1+extended darwin/amd64 BuildDate=unknown

                   | EN   
-------------------+------
  Pages            | 188  
  Paginator pages  |   0  
  Non-page files   |   0  
  Static files     | 105  
  Processed images |   0  
  Aliases          |   1  
  Sitemaps         |   1  
  Cleaned          |   0  

Built in 409 ms
Watching for changes in /Users/dom/git/test/{archetypes,content,data,layouts,static,themes}
Watching for config changes in /Users/dom/git/test/config.toml
Environment: "development"
Serving pages from memory
Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
Web Server is available at //localhost:1313/ (bind address 127.0.0.1)
Press Ctrl+C to stop
```

The page looks the following but the actual diagramm was not rendered.

<img width="996" alt="Screen Shot 2021-11-06 at 3 47 31 PM" src="https://user-images.githubusercontent.com/12104291/140613798-85befe2e-038b-4136-a0ea-dbe2e67f3d9e.png">

I observed the following error in the console in Chromium:

<img width="542" alt="Screen Shot 2021-11-06 at 3 48 16 PM" src="https://user-images.githubusercontent.com/12104291/140613803-562aef90-57d8-4df0-84d6-803623697d13.png">

Any suggestions?

**Comment by xoxys** [comment]
I'm not able to reproduce it using the described way. So far no idea what could be wrong. The root issue seems to be that the mermaid source file (mermaid-beaf30d1e1.min.js) can't be loaded. Can you open this file in an editor and check the content? It should look like a minified JS file (no need to check every line, just check if there is something in there that looks suspicious). You can also try to re-download the theme release tarball, maybe parts were corrupted somehow.

**Comment by nodeg** [comment]
You are right. I totally missed to check if the files really exist and indeed the correct Mermaid file is missing.
<img width="619" alt="Screen Shot 2021-11-07 at 2 55 30 PM" src="https://user-images.githubusercontent.com/12104291/140648004-2f9f41af-21de-4309-8a60-592ace4a7c28.png">

I did re-download the tarball and now it works. I do not know why it missed some files in the first place since I did exactly the same. Thank you for the quick response @xoxys!
